test/Microsoft.AspNetCore.SignalR.Tests/MapSignalRTests.cs

```diff
@@ -23,6 +23,20 @@ public void MapSignalRFailsForInvalidHub()
             Assert.Equal("Duplicate definitions of 'OverloadedMethod'. Overloading is not supported.", ex.Message);
         }
 
+        [Fact]
+        public void NotAddingSignalRServiceThrows()
+        {
+            var ex = Assert.Throws<InvalidOperationException>(() =>
```

Comment:
Wrap the `Assert.Throws` directly around the `UseSignalR` call, since that's what we expect to throw.